app/models/foreman_openscap/arf_report.rb

```diff
@@ -192,5 +201,22 @@ def self.references_links(references)
       end
       html_links.join(', ')
     end
+
+    def self.check_msg_for_changes(msg, incoming_data)
+      mapping = { :severity => :severity,
+                  :description => :description,
+                  :rationale => :rationale,
+                  :scap_references => :references,
+                  :value => :title }
+      mapping.each do |msg_attr, data_key|
+        msg.public_send("#{msg_attr}=".to_sym, incoming_data[data_key]) unless msg.public_send(msg_attr) == incoming_data[data_key]
```

Comment:
If I didn't check before assignment, `msg.changed?` two lines below would be always `true`. I do not want to update message if there was no change.

```ruby
msg.title #=> "some title"
msg.changed? #=> false
msg.title = "some title"
msg.changed? #=> true
```